server/__tests__/server.test.js

```diff
@@ -0,0 +1,49 @@
+import { Mockgoose } from 'mockgoose';
+import mongoose from 'mongoose';
+import server from '../../server';
+import User from '../models/user';
+
+// const request = require('supertest');
+const session = require('supertest-session');
+const verifiedUser = require('../__mockData__/verifiedUser.json');
+
+const mockgoose = new Mockgoose(mongoose);
+
+var testSession = session(server, {
+  before: function (req) {
+    req.set('user', {username: 'no-stack-dub-sack'});
```

Comment:
Didn't read the docs super thoroughly on `supertest-session` as this was a last ditch effort right before giving up for the night, but what I got from what I did read is that this is used to fake authentication and to add keys to the `req` object. However, in trying to log `req.user` in the route, I was still getting `undefined`. Tried a couple of other things briefly that did not work.